### PR TITLE
Add linux bridge port vlan API documentation

### LINKED ISSUE #39: Document vlan filtering feature
Information on how to configure vlan filtering on linux bridges is missing from the docs.

Examples on how to configure it for the most common use cases should be added, e.g. :
  - configure trunk port
  - configure access port 
  - configure trunk port w/ native vlan
  - remove vlan config

### PR DESCRIPTION
**Pull request by maiqueb**
Fixes: #39

### CODE REVIEW

File: devel/api.md [hunk 1]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
...
+            LinuxBridge.Port.TrunkTags.ID_RANGE: {
+                LinuxBridge.Port.TrunkTags.MIN_RANGE: 500,
+                LinuxBridge.Port.TrunkTags.MAX_RANGE: 599,
+            }
+        ]
+    }
+}
+```
+
+### `LinuxBridge.Port.VLAN.MODE`
```
Comment (cathay4t) [inline devel/api.md]: Should `LinuxBridge.Port.Vlan.MODE`.

File: devel/api.md [hunk 2]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
...
+
+Type: `string`
+
+The operational mode of the port. Allowed values are:
+* trunk
+* access
```
Comment (cathay4t) [inline devel/api.md]: Should be `LinuxBridge.Port.Vlan.Mode.ACCESS` and `LinuxBridge.Port.Vlan.Mode.TRUNK`.

File: devel/api.md [hunk 3]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
...
+
+### `LinuxBridge.Port.VLAN.ENABLE_NATIVE`
```
Comment (cathay4t) [inline devel/api.md]: Should be `Vlan`

File: devel/api.md [hunk 4]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
...
+
+Type: `bool`
+Default: `false`
+
+Can only be used for trunk ports. When enabled, it configures the VLAN specified by
+[`LinuxBridge.Port.VLAN.TAG`](#linuxbridgevlan_tag) as a native VLAN.
+
+### `LinuxBridge.Port.VLAN.TAG`
```
Comment (cathay4t) [inline devel/api.md]: Should be `Vlan`

File: devel/api.md [hunk 5]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
...
+
+Type: `integer`
+
+The access tag of the access port, or the native vlan of the trunk port.
+
+### `LinuxBridge.Port.VLAN.TRUNK_TAGS`
```
Comment (cathay4t) [inline devel/api.md]: Should be `Vlan`

File: devel/api.md [hunk 6]
```diff
@@ -885,6 +885,62 @@ Type: `integer`
 
 The Spanning Tree Protocol (STP) port cost for destinations via this port.
 
+## `LinuxBridge.Port.VLAN_SUBTREE`
+Each Linux Bridge port also features a subtree to configure the allowed
+VLANs passing through it. If the VLAN subtree is not configured, all the VLANs
+are allowed; once the subtree is defined, it will behave as a white-list.
```
Comment (cathay4t) [inline devel/api.md]: Not sure `white-list` means here, can you elaborate a little bit?

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: devel/api.md (hunk 6 above)
Comment (maiqueb) [reply devel/api.md]: I thought white-list was universally understood, but sure, I can.

File: devel/api.md (hunk 1 above)
Comment (maiqueb) [reply devel/api.md]: Done

File: devel/api.md (hunk 2 above)
Comment (maiqueb) [reply devel/api.md]: Done

File: devel/api.md (hunk 3 above)
Comment (maiqueb) [reply devel/api.md]: Done

File: devel/api.md (hunk 4 above)
Comment (maiqueb) [reply devel/api.md]: Done

File: devel/api.md (hunk 5 above)
Comment (maiqueb) [reply devel/api.md]: Done

**Review by cathay4t — APPROVED** [review]

**Pull request merged**